README.md

```diff
@@ -26,3 +26,5 @@ Please complete the following by November 1st. This should be the next time we w
 * Trey Watford is Bester
 * Will Dixon Is better
 * Chris Bracey Is betterer
+* Kyle Heath Is bettererer
```

Comment:
I took out the HEAD line, and the ----- line in README.md, was there something else that needed to be removed?